CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 Changes
 =======
+
+# 1.0.0 / 01-07-2020
+### Details
+https://github.com/jenkinsci/datadog-plugin/compare/datadog-0.7.1...datadog-1.0.0
+
+### Changes
+[BACKWARDS INCOMPATIBILITY NTOES]
+* Instead of sending `null` as default value for some tags we now send `unknown`
+* Event messages got changed.
+* Node tag is set by default
+
+[BUGFIX] Fix rate submission. See [179](https://github.com/DataDog/jenkins-datadog-plugin/pull/179) #TODO Confirm if the issue fixed in this PR was in the last release or not
+
+[IMPROVEMENT] Add note about how to send data to EU [154](ttps://github.com/DataDog/jenkins-datadog-plugin/pull/134)
+[IMPROVEMENT] Send delivery KPIs. See [132](https://github.com/DataDog/jenkins-datadog-plugin/pull/132) & [156](https://github.com/DataDog/jenkins-datadog-plugin/pull/156) (Thanks @pgarbe)
+[IMPROVEMENT] Add more granular statuses (i.e. `Not Built`, etc) to onCompleted event. See [153](https://github.com/DataDog/jenkins-datadog-plugin/pull/153) (Thanks @mbaitelman)
+[IMPROVEMENT] Lint project and remove unused code. See [160](https://github.com/DataDog/jenkins-datadog-plugin/pull/160)
+[IMPROVEMENT] Overall code refactor and cleanup. See [161](https://github.com/DataDog/jenkins-datadog-plugin/pull/161)
+[IMPROVEMENT] Add thread safety to dogstatsd submissions, adds `jenkins.job.started` and `jenkins.scm.checkout`, and add resiliency around potential build failures. See [169](https://github.com/DataDog/jenkins-datadog-plugin/pull/169)
+[IMPROVEMENT] Allow multiple values for tags & Support prefix for White/Blacklist & set Node tag by default & Added API + Target URL validations. See [172](https://github.com/DataDog/jenkins-datadog-plugin/pull/172)
+[IMPROVEMENT] Add 1 minute request timeout. See [174](https://github.com/DataDog/jenkins-datadog-plugin/pull/174) (Thanks @Mischa-Alff)
+[IMPROVEMENT] Add SVN tag support. See [175](https://github.com/DataDog/jenkins-datadog-plugin/pull/175)
+[IMPROVEMENT] Collect env variables and small code refactor. See [176](https://github.com/DataDog/jenkins-datadog-plugin/pull/176)
+[IMPROVEMENT] Refactor code that handles configs. See [177](https://github.com/DataDog/jenkins-datadog-plugin/pull/177)
```

Comment:
Improve plugin config. This result in one of the backward incompatible changes (the one related to script config setups)